client/js/components/procedure/StatementSegmentsList/StatementMeta/StatementPublicationAndVoting.vue

```diff
@@ -349,7 +360,6 @@ export default {
 
     save () {
       this.saveStatementVote()
-      this.$emit('save', this.localStatement)
```

Comment:
We need this emit for the anonym votes (it is a direct attribute of statement)

Reply:
ah. ok. If we restore this, we hve to ensure, that the voters-relationship is not send. Otherwise the patch request will fail because voters are not in a direct relationship from the statement

Reply:
The relationship is only sent in the request if something changed there. To ensure it is always up to date I implemented the watch for statement. Did you still have problems with this?
`  watch: {
    statement: {
      handler() {
        this.setInitValues();
      },
      deep: true
    }
  },`

Reply:
Done in [c0b79e7](https://github.com/demos-europe/demosplan-core/pull/3996/commits/c0b79e7451682f26c20250b6acacda9da775ba68)